exe/Parser/Text.hs

```diff
@@ -586,8 +586,8 @@ generateModule infile transformLines outdir modName fldGen =
     outfiledir = dirFromFileName outfile
     action c = createDirectoryIfMissing True outfiledir >> writeFile outfile c
 
-propListTransformer :: (IsStream t, Monad m) => t m String -> t m PropertyLine
-propListTransformer =
+propertyTransformer :: (IsStream t, Monad m) => t m String -> t m PropertyLine
+propertyTransformer =
```

Comment:
better to name it as `transformProperty` as `transformer` has a different connotation. Also, since it is a function its preferable to use a verb.

Reply:
Naming it to parsePropertyLines